### PR TITLE
Feature/cloud 450 add ecr create

### PR DESCRIPTION
**Pull request by rchengvariant**
# Description

Created ecr_create() function in collection.sh and is called in [entrypoint.sh in actions-dotnet](https://github.com/variant-inc/actions-dotnet/blob/feature/CLOUD-450-add-ecr-create/entrypoint.sh) and in [entrypoint in actions-nodejs](https://github.com/variant-inc/actions-nodejs/blob/master/entrypoint.sh).

Feature [CLOUD-450](https://usxtech.atlassian.net/browse/CLOUD-450)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [x] Sanity Testing
- ECR Repo Created 
  - https://github.com/variant-inc/demo-app/runs/2832777981?check_suite_focus=true
- ECR Repo Already Exists
  - https://github.com/variant-inc/demo-app/runs/2832711961?check_suite_focus=true

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet)
+Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet).
```
Comment (vibindaniel) [inline README.md:5]: Won't this be added to actions-nodejs?

**Review by vibindaniel — CHANGES_REQUESTED** [review]

File: collections.sh [hunk 2]
```diff
@@ -1,14 +1,57 @@
 #!/bin/bash
 set -e
 
-docker_publish()
+ecr_create()
 {
```
Comment (vibindaniel) [inline collections.sh:5]: Increase the indent all the contents inside flower brackets

File: collections.sh [hunk 3]
```diff
@@ -1,14 +1,57 @@
...
+            \"repositoryName\": \"$ECR_REPO\"
+        }
+    }" \
+   "$LAZY_API_URL/profiles/prod/regions/$AWS_REGION/ecr/repo"
+
+
+fi
 }
 
+
```
Comment (vibindaniel) [inline collections.sh:56]: Squash some commits. It shouldn't have more than 10 commits.

**Review by vibindaniel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rchengvariant) [reply README.md:5]: Yes it will, I will update the description.

**Pull request closed without merging**